app/javascript/components/Groceries/components/GroceryItem.tsx

```diff
@@ -45,7 +45,7 @@ export default class GroceryItem extends React.Component<GroceryItemProps> {
           {this.props.name}
         </Col>
         <Col>
-          {this.props.price}
+          ${this.props.price}
```

Comment:
In future we will have this in our internationalization work. But for now hardcoding is fine.

Reply:
+1